File: tests/fixtures/simple.go
package main

import "fmt"

func greet(name string) string {
	return "Hello, " + name
}

func add(a int, b int) int {
	return a + b
}

func multiply(x int, y int) int {
	return x * y
}

type Point struct {
	X float64
	Y float64
}

type Rectangle struct {
	Width  float64
	Height float64
}

func (r Rectangle) area() float64 {
	return r.Width * r.Height
}

File: tests/fixtures/simple.c
#include <stdio.h>

int add(int a, int b) {
    return a + b;
}

int multiply(int x, int y) {
    return x * y;
}

typedef struct {
    float x;
    float y;
} Point;

void greet(const char *name) {
    printf("Hello, %s\n", name);
}
